ppml/trusted-big-data-ml/python/docker-graphene/Makefile

```diff
@@ -27,7 +27,6 @@ all: bash.token
 endif
 
 include $(GRAPHENEDIR)/Scripts/Makefile.configs
-PYTHONVERSION = python3.6
```

Comment:
Do we need to change to python 3.7?

Reply:
It doesnot work. I think we can just delete it.